db/migrate/20200623122324_create_events.rb

```diff
@@ -0,0 +1,9 @@
+class CreateEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :events do |t|
+      t.references :user, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
#### Associations

##### User as event creator

 * Migrations
    * [ ] events table has creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

```ruby
def change
  create_table :users do |t|
  end

  create_table :events do |t|
    t.integer :creator_id
  end
end
```